test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -0,0 +1,135 @@
+            string loadGenImage = Context.Current.LoadGenImage.Expect(() => new ArgumentException("loadGenImage parameter is required for TempFilter test"));
+            string relayerImage = Context.Current.RelayerImage.Expect(() => new ArgumentException("relayerImage parameter is required for TempFilter test"));
+
+            const string trcModuleName = "testResultCoordinator";
+            const string loadGenModuleName = "loadGenModule";
+            const string relayerModuleName = "relayerModule";
+            const string trcUrl = "http://" + trcModuleName + ":5001";
+
+            string trackingId = Guid.NewGuid().ToString();
+
+            Action<EdgeConfigBuilder> addInitialConfig = new Action<EdgeConfigBuilder>(
+                builder =>
+                {
+                    // This test uses the TestResultCoordinator. It was originally designed for connectivity tests, so many of the parameters
+                    // are unnecessary for the e2e tests.
+                    // TODO: Make TestResultCoordinator more generic, so we don't have to fill out garbage values in the e2e tests.
+                    builder.AddModule(trcModuleName, trcImage)
+                       .WithEnvironment(new[]
+                       {
+                           ("trackingId", trackingId),
+                           ("eventHubConnectionString", "Unnecessary"),
+                           ("IOT_HUB_CONNECTION_STRING", this.iotHub.IoTHubConnectionString),
+                           ("logAnalyticsWorkspaceId", "Unnecessary"),
+                           ("logAnalyticsSharedKey", "Unnecessary"),
+                           ("logAnalyticsLogType", "Unnecessary"),
+                           ("testStartDelay", "00:00:00"),
+                           ("testDuration", "00:20:00"),
+                           ("verificationDelay", "00:00:00"),
+                           ("STORAGE_ACCOUNT_CONNECTION_STRING", "Unnecessary"),
+                           ("NetworkControllerRunProfile", "Online"),
+                           ("TEST_INFO", "key=unnecessary")
+                       })
+                       .WithSettings(new[] { ("createOptions", "{\"HostConfig\": {\"PortBindings\": {\"5001/tcp\": [{\"HostPort\": \"5001\"}]}}}") })
+
+                       .WithDesiredProperties(new Dictionary<string, object>
+                       {
+                           ["reportMetadataList"] = new Dictionary<string, object>
+                           {
+                               ["reportMetadata1"] = new Dictionary<string, object>
+                               {
+                                   ["TestReportType"] = "CountingReport",
+                                   ["TestOperationResultType"] = "Messages",
+                                   ["ExpectedSource"] = "loadGenModule.send",
```

Comment:
Update other parameter to refer to variable if possible.
```suggestion
                                   ["ExpectedSource"] = $"{loadGenModuleName}.send",
```